src/__main__.py

```diff
@@ -14,19 +14,19 @@ def nest_yaml(tag: str, content: str, indent_size:int=4) -> str:
 
 def log_yaml(full_yaml: str) -> None:
     # save locally
-    local_dir_path = f"models/{os.path.basename(os.path.dirname(wandb.run.dir)).replace('run-', '')}"
+    local_dir_path = f"models/{os.path.basename(os.path.dirname(wandb.run.dir)).replace('run-', '')}" # pyright: ignore [reportOptionalMemberAccess]
```

Comment:
Adding an ignore comment *every* time we use `wandb.run.dir` seems verbose ... is there a way to ensure that pyright ignores just this pattern?

Reply:
What do you think @riensou ?

Reply:
in-persons discussion agreed that this is acceptable, as it brings attention to the "need" for type stubbing (like a TODO), but is a reasonable middle ground for productivity now